src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunItems.java

```diff
@@ -56,6 +56,7 @@ private SlimefunItems() {}
     public static final SlimefunItemStack REINFORCED_CLOTH = new SlimefunItemStack("REINFORCED_CLOTH", Material.PAPER, "&bReinforced Cloth", "", "&fThis cloth has been reinforced", "&fwith &bLead &fto protect against", "&fradioactive substances");
     public static final SlimefunItemStack TIN_CAN = new SlimefunItemStack("CAN", HeadTexture.TIN_CAN, "&fTin Can");
     public static final SlimefunItemStack NIGHT_VISION_GOGGLES = new SlimefunItemStack("NIGHT_VISION_GOGGLES", Material.LEATHER_HELMET, Color.BLACK, "&aNight Vision Goggles", "", "&9+ Night Vision");
+    public static final SlimefunItemStack CRASH_HELMET = new SlimefunItemStack("CRASH_HELMET", Material.IRON_HELMET, "&5Crash Helmet", "", "&7This helmet will protect you from", "&7crashing while flying elytra.");
```

Comment:
I don't know :/ I'm not the greatest fan of the name "Crash Helmet" but I also can't think of a better one, any ideas?

Reply:
Feather Helmet or Shield Helmet? Idk

Reply:
Maybe something like Elytra Cap? And then include elytra scales in the recipe?

Reply:
hard hat, safety helmet...

Reply:
Barrete or Elytra Barette

Reply:
ima think about elytra cap (elytra scales is a great idea)